### PR TITLE
feat: apply autoscaler fixes

### PR DESCRIPTION
**Pull request by tracetechnical**
## What I am changing
I am moving the VMs to a type with a higher available quota and adding a seperate compute pool

## How I did it
I added a K8s nodepool of larger VMs for compute

## How you can test it
Run `make deploy`

### CODE REVIEW

File: terraform/cluster.tf [hunk 1]
```diff
@@ -5,17 +5,12 @@ resource "azurerm_kubernetes_cluster" "bakery_cluster" {
...
-    min_count           = 1
-    vm_size             = "standard_d4_v4"
-    os_disk_size_gb     = 512
-    enable_auto_scaling = true
-  }
-
-  auto_scaler_profile {
-    scale_down_delay_after_add = "2m"
-    scale_down_unneeded        = "2m"
+    name            = "system"
```
Comment (TomAugspurger) [inline terraform/cluster.tf:8]: Just a note that changing this *might* delete and recreate the entire kubernetes cluster. Hopefully that's OK.

Comment (tracetechnical) [reply terraform/cluster.tf:8]: It will indeed, but the code in this repo is idempotent, so this shouldn't affect much.

Comment (tracetechnical) [reply terraform/cluster.tf:8]: (IE the manifests etc, so rerunning should only yield a re-spin)

**Comment by tracetechnical** [comment]
Plot twist: this fixes nothing

**Pull request closed without merging**